### PR TITLE
Add menu-item-radio example

### PR DESCRIPTION
**Pull request by joaovitorzv**
Add `menu-item-radio` example for #939 

image

### CODE REVIEW

**Comment by diegohaz** [comment]
Thank you @joaovitorzv! I'm not sure this is a good example, though. This kind of UI (where the button displays the selected value) would be better implemented with a Select component.

Do you have any other examples in mind? I'll try to think of one too.

**Comment by joaovitorzv** [comment]
Yeah I agree, I thought about including the `menu.values.view` in the example and missed the point of it being a **menu**.

Other possible examples I can think of: 

- A "Panel position" button with bottom, left, right and top options
- A "Layout" button with default, split, horizontal and vertical options
- A "Sort" button with Popular, newest, oldest options
  
if I still don't get it you can give me an example i'll be happy to fix this!

**Comment by diegohaz** [comment]
> * A "Sort" button with Popular, newest, oldest options

That sounds good.

We can show the selected value in a separate element. Like, `Sorting by Oldest` or something.

I can see that would be a valid case in GitHub:
Screenshot of the GitHub issues page with the Sort menu open

File: packages/ariakit/src/menu/__examples__/menu-item-radio/index.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  MenuItemCheck,
+  MenuItemRadio,
+  useMenuState,
+} from "ariakit/menu";
+import "./style.css";
+
+export default function Example() {
+  const menu = useMenuState({ gutter: 8, defaultValues: { sort: "popular" } });
+
+  return (
```
Comment (diegohaz) [inline packages/ariakit/src/menu/__examples__/menu-item-radio/index.tsx]: Nit:
```suggestion
  const menu = useMenuState({
    gutter: 8,
    defaultValues: { sort: "popular" },
  });
  return (
```

File: packages/ariakit/src/menu/__examples__/menu-item-radio/style.css [hunk 2]
```diff
@@ -0,0 +1,2 @@
+@import "../../../button/__examples__/button/style.css";
+@import "../menu/style.css";
```
Comment (diegohaz) [inline packages/ariakit/src/menu/__examples__/menu-item-radio/style.css]: I think menu already imports button styles.
```suggestion
@import "../menu/style.css";
```

File: packages/ariakit/src/menu/__examples__/menu-item-radio/test.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+});
+
+test("update checked menu item on space", async () => {
+  render(<Example />);
+  await press.Tab();
+  await press.Space();
+  await press.ArrowDown();
+  await press.Space();
+  expect(getMenuItem("Newest")).toHaveAttribute("aria-checked", "true");
+});
```
Comment (diegohaz) [inline packages/ariakit/src/menu/__examples__/menu-item-radio/test.tsx:43]: Nice!

**Review by diegohaz — APPROVED** [review]
Great job @joaovitorzv!

**Pull request merged**